spec/features/diff_viewer_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require "#{Rails.root}/lib/replica"
+
+describe 'diff viewer', type: :feature, js: true do
+  let(:en_wiki) { Wiki.get_or_create(language: 'en', project: 'wikipedia') }
+  let(:course) { create(:course, id: 1, start: '2013-01-01', end: '2014-01-02') }
+  let(:user) { create(:user, id: 1, username: 'I enjoy sandwiches') }
```

Comment:
It would be useful to add some documentation to explain that what you're trying to do here... looking for actual revisions made by this user to a specific page, in particular https://en.wikipedia.org/w/index.php?title=Periodontium&diff=715718156&oldid=714831980&diffmode=source

Reply:
Done!